css/src/components/popover.scss

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable scss/at-extend-no-missing-placeholder */
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		@extend .button;
+
+		min-height: unset;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: 150px;
+		margin-top: $spacer-3;
+		padding: $spacer-6;
+		border: 1px solid $border;
+		border-radius: $border-radius;
+		background-color: $body-background;
+		box-shadow: $box-shadow-heavy;
+		z-index: $zindex-popover;
+
+		&.popover-content-align-left {
+			left: 0;
+		}
+
+		&.popover-content-align-right {
```

Comment:
I personally prefer top level classes for this type of thing. Taking the `.buttons` component as an example: `.buttons.buttons-right` is more pleasant for the developer. It will require some changing of the selectors but could instead do something like:

```scss
.popover {
  &.popover-right {
    right: 0;
  }
}
```

Reply:
I thought about that too, but the alignment should be applied on the `popover-content`, so I thought it would make more sense. But I can update to this:

```css
.popover {
  ...

  &.popover-right {
    .popover-content {
      right: 0;
    }
  }
}
```

Reply:
Updated

Reply:
Yes, perfect!